src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -25,7 +25,7 @@
  * @EntityReferenceSelection(
  *   id = "og:default",
  *   label = @Translation("OG selection"),
- *   group = "default",
+ *   group = "og",
```

Comment:
OK, so if we are doing that, then we DEFINITELY need the below change to not throw anyway :) Where would that behaviour of always selecting the of handler get used?